specification/2.0/README.md

```diff
@@ -862,7 +863,13 @@ After applying morph targets to vertex positions and normals, tangent space may
 
 ### Skins
 
-All skins are stored in the `skins` array of the asset. Each skin is defined by the `inverseBindMatrices` property (which points to an accessor with IBM data), used to bring coordinates being skinned into the same space as each joint; and a `joints` array property that lists the nodes indices used as joints to animate the skin. The order of joints is defined in the `skin.joints` array and it must match the order of `inverseBindMatrices` data. The `skeleton` property (if present) points to the node that is the common root of a joints hierarchy or to a direct or indirect parent node of the common root.
+All skins are stored in the `skins` array of the asset. Each skin is defined by the `inverseBindMatrices` property (which points to an accessor with IBM data), used to bring coordinates being skinned into the same space as each joint; and a `joints` array property that lists the nodes indices used as joints to animate the skin. The order of joints is defined in the `skin.joints` array and it must match the order of `inverseBindMatrices` data.
+
+All joints defined in a skin must form a connected subtree within the node hierarchy, and the highest order node is the `skin parent`. If a skin contains multiple distinct subtrees (multi-rooted, a joint is not defined which connects them) then the "roots" of the distict subtrees must be siblings in the node hierarchy, and their comment parent is the `skin parent` (note that in this case, the `skin parent` may not be a joint).
```

Comment:
Given the motivation for these restrictions, why allow multiple distinct subtrees at all? Unless I've missed something, couldn't any skin with multiple distinct subtrees and a common parent be made into a single connected subtree, by turning that common parent into a joint?